consensus/src/block_storage/sync_manager.rs

```diff
@@ -489,12 +489,8 @@ impl BlockRetriever {
                         e,
                     );
                     // select next peer to try
-                    if peers.is_empty() {
-                        bail!(
-                            "Failed to fetch block {} in {} attempts: no more peers available",
-                            block_id,
-                            attempt
-                        );
+                    if peers.is_empty() || attempt >= 10 {
+                        bail!("Failed to fetch block {} in {} attempts", block_id, attempt);
                     }
```

Comment:
Should you log that one is over the retry limit, vs one is that it has no peers?

Reply:
we have more than 200 peers in AIT3 so it'll always be over retry limit